Copia 11 i

alcamk opened the pull request:
# Description
- Complete Sign Up screen design
- Buttons, checkboxes and text fields with normal functionality
- Creation of checkbox component 
- Text with link option functionality 
- This is linked to #11 

 
 Examples:
- You can write in the text fields and check the color change of the border
- You can click the eye icon and check for password hidden functionality 
 
 # Testing
Go to Reservaciones/ folder and run $ npm install
Go to ios/ folder and run $ pod install
Go back to the project folder $ cd ..
Run $ npx react-native start
Run $ npx react-native run-ios for ios o $ npx react-native run-android for android 
  
  ## Screenshots 
![captura3 png](https://user-images.githubusercontent.com/58040748/99595021-9e82e880-29b9-11eb-9bb8-c1ac170d2054.png)
![captura6](https://user-images.githubusercontent.com/58040748/99595027-9fb41580-29b9-11eb-967a-0a8ee5fea51f.png)
![captura5](https://user-images.githubusercontent.com/58040748/99595032-a0e54280-29b9-11eb-81d0-855418e5997a.png)
![captura1](https://user-images.githubusercontent.com/58040748/99595034-a0e54280-29b9-11eb-9e6d-01d2f5010f2a.png)
![captura2](https://user-images.githubusercontent.com/58040748/99595035-a17dd900-29b9-11eb-8ca8-26cdf2635df6.png)
![captura4](https://user-images.githubusercontent.com/58040748/99595036-a2166f80-29b9-11eb-9a7d-fe792d58f904.png)

alcamk committed:
Signup rebase project

Creation of a new branch and copied all the files from existing one, this was made to fix problems
of exceded number of files from node_modules and other carpets

alcamk committed:
Textfield component

Text field details working, this functionality only puts de border in diferent color when is focus.

JordiGN reviewed (dismissed):
nice one adding the const for texts and colors

luisarmandodv commented on Reservaciones/components/TextField.js:
```diff
@@ -0,0 +1,52 @@
+
+
+TextField.propTypes = {
+  title: PropTypes.string,
+  Hfocus:  PropTypes.func,
+  HBlur:  PropTypes.func,
+  campo: PropTypes.string,
+  keyboard : PropTypes.bool,
+  errorLogin: PropTypes.string,
+  changeText: PropTypes.func
```
`nameHandlerFocus` not defined as a PropType

luisarmandodv commented on Reservaciones/components/CheckboxCom.js:
```diff
@@ -0,0 +1,69 @@
+}
+
+class  ComCheckbox extends React.Component {
+    constructor(props) {
+        super(props);
+
+      }
```
pointless constructor

luisarmandodv commented on Reservaciones/components/CheckboxCom.js:
```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+
+
+const checked = () => {
+    console.log('Checked');
+}
```
unused arrow function

luisarmandodv commented on Reservaciones/components/CheckboxCom.js:
```diff
@@ -0,0 +1,69 @@
+import checkboxStyle from '../Styles/CheckboxStyle';
+import { Texts } from '../ContentText/TextsSignup';
+import {Colors} from '../Styles/Colors';
+import PropTypes from 'prop-types';
```
global packages goes before local importing.

luisarmandodv commented on Reservaciones/components/Button.js:
```diff
@@ -0,0 +1,42 @@
+
+
+class ButtonAction extends Component {
+  constructor(props) {
+    super(props);
+  }
```
```suggestion
```

luisarmandodv replied:
what about this? @alcamk

luisarmandodv commented on Reservaciones/components/Button.js:
```diff
@@ -0,0 +1,42 @@
+}
+
+ButtonAction.propTypes = {
+  imageRequired: PropTypes.bool,
+  title: PropTypes.string,
+  Press : PropTypes.func
```
`stateComponent` prop missing to be defined

alcamk replied:
@Ocosmee con you help me in this change pleas!

luisarmandodv commented on Reservaciones/Styles/SignupStyle.js:
```diff
@@ -0,0 +1,22 @@
+        marginTop: 25,
+        marginBottom: 25,
+    },
+    
```
```suggestion
```

luisarmandodv commented on Reservaciones/Screens/Signup.js:
```diff
@@ -0,0 +1,133 @@
+        return(
+            <View style={SignupStyle.Father}>
+                <Text style={SignupStyle.TextTittle}>{Texts.signup}</Text>
+                <TextField
+                    title={Texts.firstname}
+                    nameHandlerFocus={this.state.Vfirstname}
+                    Hfocus={() => this.handlerFocus(Texts.Pfirstname)}
+                    HBlur={() => this.handlerBlur(Texts.Pfirstname, this.state.userFirstName)}
+                    campo= 'userFirstName'
+                    keyboard = {false}
+                    errorLogin=''
+                    changeText={this.handleChangeText}
+                  />
+                    <TextField
+                    title={Texts.email}
+                    nameHandlerFocus={this.state.Vemail}
+                    Hfocus={() => this.handlerFocus(Texts.Pemail)}
+                    HBlur={() => this.handlerBlur(Texts.Pemail, this.state.userEmail)}
+                    campo= 'userEmail'
+                    keyboard = {false}
+                    errorLogin=''
+                    changeText={this.handleChangeText}
+                  />
+                    <TextField
+                    title={Texts.pass}
+                    nameHandlerFocus={this.state.Vpassword}
+                    Hfocus={() => this.handlerFocus(Texts.Ppassword)}
+                    HBlur={() => this.handlerBlur(Texts.Ppassword, this.state.userPass)}
+                    campo= 'userPass'
+                    keyboard = {false}
+                    errorLogin=''
+                    changeText={this.handleChangeText}
+                  />
```
indentation here is wrongly implemented,

luisarmandodv commented on Reservaciones/Screens/Signup.js:
```diff
@@ -0,0 +1,133 @@
+
+
+class Signup extends React.Component{
+    constructor(props) { 
+        super(props); this.state = {
+             loading: false, 
+             isPasswordHidden: true, 
+             userEmail: '', 
+             userPass: '', 
+             errorLogin: '',
+             userFirstName: '',
+            };
+        }
+
```
indentation here can be improved

luisarmandodv commented on Reservaciones/Screens/Signup.js:
```diff
@@ -0,0 +1,133 @@
+import React, {Component} from 'react';
+import {View, Text, TextInput} from 'react-native';
+import SignupStyle from '../Styles/SignupStyle';
+import PropTypes from 'prop-types';
```
global importing goes before local importing

alcamk committed:
Requested changes fixed

Making correction to improve the code style and good practice with propsStill waiting for stateComponent fix by other team member

alcamk committed:
PR Issue 11 changes fixed
All requested changes fixed

luisarmandodv commented on Reservaciones/Screens/Signup.js:
```diff
@@ -0,0 +1,148 @@
+
+
+
+
```
```suggestion
```

luisarmandodv commented on Reservaciones/components/Button.js:
```diff
@@ -0,0 +1,45 @@
+import {View, Text, TouchableOpacity,Image} from 'react-native';
+import PropTypes from 'prop-types';
+import styleLogin from '../Styles/LoginStyle';
+
```
```suggestion
```

luisarmandodv commented on Reservaciones/Styles/SignupStyle.js:
```diff
@@ -0,0 +1,21 @@
+        textAlign: 'left',
+        marginTop: 25,
+        marginBottom: 25,
+    },
```
indentation has 4 spaces, but all the project I see it has 2 spaces.

luisarmandodv commented on Reservaciones/Styles/Colors.js:
```diff
@@ -0,0 +1,10 @@
+    darkGray: '#21252B',
+    transparent: '#00000040',
+  };
+  
```
```suggestion
```

luisarmandodv commented on Reservaciones/Styles/CheckboxStyle.js:
```diff
@@ -0,0 +1,22 @@
+        color: '#AFB2B7',
+        width: '100%',
+    }
+
```
```suggestion
```

luisarmandodv commented on Reservaciones/Styles/CheckboxStyle.js:
```diff
@@ -0,0 +1,22 @@
+
+const checkboxStyle = StyleSheet.create({
+    container: {
+        marginBottom: 20,
+        width: 30,
+        height: 30,
+        flexDirection: "row",
+        backgroundColor:'blue'
+    },
+    item: {
+        marginTop: 60,
+        marginRight: 10,
+    },
+    txtChkbx: {
+        color: '#AFB2B7',
+        width: '100%',
```
indentation has 4 spaces, but all the project I see it has 2 spaces.

luisarmandodv commented on Reservaciones/Screens/Signup.js:
```diff
@@ -0,0 +1,148 @@
+    stateComponent: PropTypes.object
+  };
+
+export default Signup;
```
```suggestion
export default Signup;

```

luisarmandodv commented on Reservaciones/ContentText/TextsSignup.js:
```diff
@@ -0,0 +1,24 @@
+    agreeCheckBox: 'agreeCheckBox',
+    subscribeCheckBox: 'subscribeCheckBox',
+  };
+  
```
```suggestion
```

alcamk committed:
Changes sugested fixed
